src/domain/event/EventUtils.ts

```diff
@@ -78,7 +78,7 @@ export const formatPrice = (price?: string): string => {
     return '';
   }
 
-  const priceRegex = /^\d+([/\-.,]{1})?\d+?$/;
+  const priceRegex = /^\d+([/\-.,]{1}\d+)?$/;
```

Comment:
https://regex101.com/ says: `{1} Quantifier — Matches exactly one time (meaningless quantifier)`
so this {1} is probably redunant and could be removed,

Reply:
true

Reply:
fixed